packages/did/README.md

```diff
@@ -28,7 +28,7 @@ const publicKey = '-----BEGIN PUBLIC KEY-----//..-----END PUBLIC KEY-----';
 const { did, document, seed } = await register({ seed, network: '0x1', publicKey });
 ```
 
-#### Register Identity with Specific Nodes
+#### Register Identifier with Specific Nodes
 
 ```javascript
 import { register, IdenityRegistry } from '@tangleid/did';
```

Comment:
Like here, the description and the function name maybe will make people misunderstand.

Reply:
I think `IdenityRegistry` is used to register/resolve the `identifier` for the `digital identity`, so the `IdenityRegistry ` is fine.
I changed the descriptions, you could check out is it better or not.
If it is still confusing, we could rename it. 😃

Reply:
Change `IRI` to `IOTA nodes`. IRI is a program, but nodes empower IOTA network.

Reply:
Got it. @mushroom0117 is it still confusing?

Reply:
@ender503 No, it's much better to understand. 👍